tests/integration/Mailer/MailerTest.php

```diff
@@ -208,7 +208,10 @@ public function testItCanSendWhenSubscriberEntityIsPassed() {
     }
 
     $subscriberFactory = new SubscriberFactory();
-    $subscriber = $subscriberFactory->withFirstName('Recipient')->create();
+    $subscriber = $subscriberFactory
+      ->withEmail('mailpoet-phoenix-test@mailinator.com')
```

Comment:
I used mailpoet-phoenix-test@mailinator.com instead of blackhole@mailpoet.com because the other sending tests use the former, and those are all getting delivered fine. If we don't care about deliverability then we might want to change this to `blackhole@mailpoet.com` in both places. What do you think?

Reply:
I would prefer our blackhole email address because we don't care about deliverability in this case.

Reply:
I just pushed an update that uses the blackhole@mailpoet.com address for both this test and the other sending test in the same file.

Are we actively checking emails sent to mailinator anywhere, or were we just using it because we didn't have the blackhole address set up yet? I'm seeing the mailpoet-phoenix-test@mailinator.com address used in the specific mailer method tests (`AmazonSESTest`, `MailPoetAPITest`, `PHPMailTest`, `SendGridTest`, and `SMTPTest`), and I'm wondering if it would be okay to update those to use the blackhole as well. At first glance it looks like that would be a safe change to make.

Reply:
@johnolek I think that we can replace the `mailpoet-phoenix-test@mailinator.com` address. The blackhole email is supported from July 2021.

Reply:
I believe I've now replaced the mailinator address with the blackhole address everywhere @lysyjan